uds-package.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: uds.dev/v1alpha1
+kind: Package
+metadata:
+  name: podinfo
+  namespace: podinfo
+spec:
+  network:
+    expose:
+      - service: podinfo
+        podLabels:
+          app: podinfo
+        host: podinfo
+        gateway: tenant
+        port: 9898
```

Comment:
Since you are doing this can you add a wait for network on this port to test it works?

Reply:
done